### ISSUE TITLE
Data not persistant

### ISSUE THREAD

**Issue by Azazel666** [docker-library/docs#1285]
I have a issue with my data is getting lost each time i recreate my mongo db container in docker.
I have a wiki.js container wich saves data in mongo and i can recreate the wiki.js container without any problems but if i recreate the mongo db for some reason i lose everything in the wiki.js site.

Here is my mongo setup in docker compose:
`mongodb:
    container_name: mongodb
    image: mongo
    restart: always
    expose:
      - '27017'
    command: '--smallfiles --bind_ip mongodb'
    environment:
      - PUID=${PUID}
      - PGID=${PGID}
      - TZ=${TZ}
      - 'MONGO_LOG_DIR=/dev/null'
    volumes:
      - ${USERDIR}/mongo/data/db:/data/db
      - /etc/localtime:/etc/localtime:ro
    networks:
      - traefik_proxy`

as far as i know the /data/db should hold the data and not been reset if the container is re-created. Am i missing something ?

**Comment by wglambert** [comment]
Are your variables correctly set and interpreted? Can you view the local mountpoint of the volume and see the directory populated

**Comment by Azazel666** [comment]
Yes the local /mongo/data/db folder is filled with files, mostly with extension .wt also a directory called diagnostic.data & joural.

**Comment by Azazel666** [comment]
I dont have any issues when re-creating the wiki docker container, it works just fine. But if the mongo db container is re-created for some reason i lose the whole content of my wiki.
